src/services/services.ts

```diff
@@ -1079,6 +1079,7 @@ module ts {
 
     export interface CompletionInfo {
         isMemberCompletion: boolean;
+        isBuilder: boolean;

```

Comment:
The name of the property is not clear. i think we should name it on what it stands for instead of how it is going to be used. so something a long the lines of possibleIdentifierDefinition, possibleDeclarationLocation.. 
also a comment explaining what it does would be appreciated.